File: iniciante/python/1117.py
# -*- coding: utf-8 -*-

notas_validas = 0
soma_notas = 0
while notas_validas < 2:
    nota = float(input())
    if nota >= 0 and nota <= 10:
        soma_notas += nota
        notas_validas += 1
    else:
        print("nota invalida")

media = soma_notas / 2
print("media = %.2f" %media)

File: iniciante/c/1117.c
#include <stdio.h>
 
int main() {

	float nota, media, soma_das_notas;
	int notas_validas;

	notas_validas = 0;
	soma_das_notas = 0;
	while (notas_validas < 2){
		scanf("%f", &nota);
		if (nota < 0 || nota > 10){
			printf("nota invalida\n");
		}else{
			notas_validas += 1;
			soma_das_notas += nota;
			if (notas_validas == 2){
				media = soma_das_notas / 2;
				printf("media = %.2f\n", media);
			}
		}
	}
    return 0;
}